standard/clause_7_core.adoc

```diff
@@ -5,14 +5,17 @@
 
 include::requirements/requirements_class_core.adoc[]
```

Comment:
Maybe a little pedantic but do we want to harmonize this:
- http://www.opengis.net/spec/ogcapi-features-1/1.0/req/core
- http://www.opengis.net/spec/ogcapi-coverages/1.0/req/core
- http://www.opengis.net/spec/ogcapi_common/1.0/req/core

Reply:
Telco 20191218: Agreed to add "-1" also for coverages.